src/components/index/Hero.tsx

```diff
@@ -7,10 +7,10 @@ import { INDEX_HERO_SUBTITLE, SITE_NAME } from "../../config";
 export const Hero = memo(() => {
   return (
     <section className="relative flex flex-col items-center justify-center gap-4 pt-28">
-      <div className="-z-1 pointer-events-none absolute h-64 w-80 -translate-y-60 scale-[2.5] rounded-full from-indigo-800/60 to-indigo-800/20 blur-3xl dark:bg-gradient-to-br" />
-      <div className="-z-1 pointer-events-none absolute h-12 w-80 translate-y-60 scale-[2] rounded-full from-yellow-800/70 to-indigo-800/30 blur-3xl dark:bg-gradient-to-br" />
+      <div className="-z-1 pointer-events-none absolute h-64 w-80 md:opacity-100 -translate-y-[320px] md:-translate-y-60 scale-[2.5] rounded-full from-indigo-800/60 to-indigo-800/20 blur-3xl dark:bg-gradient-to-br" />
+      <div className="-z-1 pointer-events-none absolute h-12 w-80 md:translate-y-60 scale-[2] rounded-full from-yellow-800/70 to-indigo-800/30 blur-3xl dark:bg-gradient-to-br" />
 
-      <h1 className="lg:6xl z-20 bg-gradient-to-tl from-indigo-500 to-pink-500 bg-clip-text text-5xl font-medium leading-none tracking-wide text-transparent md:text-6xl">
+      <h1 className="lg:6xl z-20 bg-gradient-to-tl from-indigo-500 to-pink-500 bg-clip-text text-5xl font-semibold tracking-normal leading-none text-transparent md:text-6xl">
```

Comment:
I guess it should be `bold` here and bigger on bigger screens

Reply:
In my opinion, the semibold looks much clearer, but I can change it, no problem. It's changing the size from 5xl to 6xl.
<img width="1403" alt="Screenshot 2023-01-31 at 20 59 27" src="https://user-images.githubusercontent.com/78202674/215869360-ea5c9bc0-f88e-447d-ab4c-c7f9b524ef0e.png">

Reply:
👍 IMO 6xl looks better